apps/src/code-studio/header.js

```diff
@@ -93,7 +93,7 @@ header.build = function (scriptData, stageData, progressData, currentLevelId, pu
     $('.header_popup_link_glyph').html('&#x25B2;');
     $('.header_popup_link_text').text(dashboard.i18n.t('less'));
     $(document).on('click', hideHeaderPopup);
-    lazyLoadPopup();
+    progress.renderMiniView($('.user-stats-block')[0], scriptName, currentLevelId);
```

Comment:
I know jQuery isn't going away anytime soon, but would `document.querySelector()` be slightly better here since we don't use any of the jQuery wrapper features?